src/VirtoCommerce.ContentModule.Azure/AzureContentBlobStorageProvider.cs

```diff
@@ -86,10 +86,7 @@ private string NormalizeUrl(string url)
                 {
                     url = Uri.UnescapeDataString(new Uri(url).AbsolutePath);
                 }
-                result = Path.DirectorySeparatorChar + url.Replace('/', Path.DirectorySeparatorChar).TrimStart(Path.DirectorySeparatorChar);
-                result = _options.RootPath + Path.DirectorySeparatorChar + result.Replace(_options.RootPath, string.Empty);
-                //TODO: need to use Path.DirectorySeparatorChar instead of hardcoded value
-                result = Regex.Replace(result, @"\\+", $"{Path.DirectorySeparatorChar}");
+                result = UrlHelperExtensions.Combine(_options.RootPath, url.Replace(_options.RootPath, string.Empty));
```

Comment:
What this code doing?

Reply:
It's OK. It makes absolute URL for relative paths. Or do nothing if the path is absolute.